src/isar/apis/schedule/start_echo_mission.py

```diff
@@ -1,65 +1,43 @@
+
+
+
```

Comment:
Should be no newlines. Black will resolve this for you once you run it for all files. However, it is useful to be aware of this and why it should be this way even if Black will resolve it for you.